lib/sycamore/sycamore/query/operators/query_database.py

```diff
@@ -63,19 +62,15 @@ def output_type(self) -> type:
 class QueryVectorDatabase(Node):
     """Vector search to load records that are similar to a given query string. Returns the top k records
     according to the vector similarity between the query string and record's text content.
-    Use this if you need to filter on a field that can't suffice with an exact match, but also isn't complex enough to
-    require a llm_filter, i.e. a record level LLM call.
-
-    Here are some instructions about using QueryVectorDatabase instead of QueryDatabase:
-    1. Use QueryVectorDatabase if the query plan uses an LLM operator (LLMFilter or LLMExtractEntity),
-        and the final answer doesn't require you to scan the full dataset (e.g. find me an instance of a transcript
-        that talked about AI).
-    2. Use QueryVectorDatabase for RAG-style questions where you need to retrieve a set of candidate records where the
-        answer might exist.
-    3. Do not use QueryVectorDatabase when examining each record is essential (e.g. did all transcripts talk about AI?).
-    4. Do not use QueryVectorDatabase where you need to potentially return large datasets (e.g. give me all transcripts
-        that talked about AI).
 
+    You should only use QueryVectorDatabase for the following query types:
+    1. "Is there *any* <record> similar to <query>?" - yes or no questions about inclusion of any record
```

Comment:
I'm a stickler for ending sentences with periods on the (probably superstitious) belief that this helps the LLM tokenize better :-)